pkg/alerts/alertsHandler/alertsHandler.go

```diff
@@ -173,6 +181,115 @@ func ProcessSilenceAlertRequest(ctx *fasthttp.RequestCtx) {
 	utils.WriteJsonResponse(ctx, responseBody)
 }
 
+func ProcessTestContactPointRequest(ctx *fasthttp.RequestCtx) {
+	var testContactRequest TestContactPointRequest
+	if err := json.Unmarshal(ctx.PostBody(), &testContactRequest); err != nil {
+		utils.SendError(ctx, "Failed to unmarshal json", "ProcessTestContactPointRequest: Request Body: "+string(ctx.PostBody()), err)
+		return
+	}
+
+	switch testContactRequest.Type {
+	case "slack":
+		channelID, ok := testContactRequest.Settings["channel_id"].(string)
+		if !ok {
+			utils.SendError(ctx, "channel_id is required but is missing", "ProcessTestContactPointRequest: Slack Channel ID is missing. Request Body: "+string(ctx.PostBody()), nil)
```

Comment:
nit: In `SendError()` you also don't have to repeat `"Slack Channel ID is missing"` in the third parameter because you've included it in the second parameter as `"channel_id is required but is missing"`. You can look at the definition of `SendError` in httpserverutils.go